src/main/java/seedu/address/logic/commands/contact/CDeleteCommand.java

```diff
@@ -20,14 +20,14 @@
 public class CDeleteCommand extends Command implements Undoable {
 
     public static final String COMMAND_WORD = "cdelete";
-    public static final String PARAMETERS = "INDEX[-INDEX]\n";
+    public static final String PARAMETERS = "INDEX1[-INDEX2]";
     public static final String SYNTAX = COMMAND_WORD + " " + PARAMETERS;
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the contact identified by the index number used in the displayed contact list."
```

Comment:
We can add \n before listing parameters, to standardize the message_usage across commands